addon/-private/utils/element/estimate-element-height.js

```diff
@@ -1,48 +1,31 @@
 import { assert } from '@ember/debug';
 
-export function estimateElementHeight(element, fallbackHeight) {
+export default function estimateElementHeight(element, fallbackHeight, scale) {
+  assert(`You called estimateElement height without a scale`, scale);
   assert(`You called estimateElement height without a fallbackHeight`, fallbackHeight);
   assert(`You called estimateElementHeight without an element`, element);
 
   if (fallbackHeight.indexOf('%') !== -1) {
-    return Math.max(getPercentageHeight(element, fallbackHeight), 1);
+    return getPercentageHeight(element, fallbackHeight, scale);
   }
 
   if (fallbackHeight.indexOf('em') !== -1) {
-    return Math.max(getEmHeight(element, fallbackHeight), 1);
+    return getEmHeight(element, fallbackHeight);
   }
 
-  // px or no units
-  return Math.max(parseInt(fallbackHeight, 10), 1);
+  return parseInt(fallbackHeight, 10);
```

Comment:
does this fix a bug where we no longer need to ensure a min height of 1? 🎉

Reply:
ah, actually we still need to define some kind of behavior here, since our pool size is determined by `scrollContainerHeight / estimateHeight`. 

I think we could default `estimateHeight` to `1px` or something, but it would probably be a better idea to assert. Either way, I don't think that belongs in `estimateElementHeight`